DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Utility.cs

```diff
@@ -28,6 +28,9 @@ public static class Utility
                 { "D", AgeType.Day },
             };
 
+        public static readonly int AgeStringLength = 3;
```

Comment:
Soft Suggestion: **const** is better than **static readonly** here
It's a real constant value, won't be changed anywhere, any time.